pom.xml

```diff
@@ -163,28 +163,28 @@
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>javax.enterprise</groupId>
-                <artifactId>cdi-api</artifactId>
+                <groupId>jakarta.enterprise</groupId>
+                <artifactId>jakarta.enterprise.cdi-api</artifactId>
```

Comment:
So, should we exclude the EJB api from here?
Metrics currently do this https://github.com/eclipse/microprofile-metrics/blob/2.3-M3/pom.xml#L113
Or we can push Jakarta guys to release `2.0.3` with the transitive dependencies fixed

Reply:
Personally, I prefer the latter. Do you also agree, and can we then initialise actions to get it fixed in time for MP 3.3?

Reply:
Thanks, @jmartisk for the pointer.  I agree that we should exclude ejb and at the same time, pressure cdi to correct this.  Are you able to help push this along to the cdi team?  If they can correct this in a 2.0.3 release before we finalize MP 3.3, then we can remove the exclusion.

I also notice that Metrics is excluding EL and Annotations.  We bring in Annotations anyway as a dependency, but should we also be excluding EL at the platform level?  I think we should.  Comments?

Reply:
I also think we should exclude EL at the platform level since we have a template engine (like JSP) available in MP that uses it, right?

Reply:
The root problem is https://github.com/eclipse-ee4j/interceptor-api/issues/31. We need to release interceptor first and then update CDI to use the latest version of interceptor to remove the ejb dependency.

Reply:
EL should be excluded IMO, reasoning is in this discussion: https://github.com/eclipse/microprofile/issues/56

Reply:
EL should be excluded IMO, reasoning is in this discussion: https://github.com/eclipse/microprofile/issues/56

Reply:
Regarding the EJB api fix, I asked Arjan on https://github.com/eclipse-ee4j/interceptor-api/issues/31 if we can get Interceptors 1.2.6 out, after that we will need an updated CDI artifact as well

Reply:
If we go this route with the Jakarta EE 8 dependencies in MP 3.3, we won't have the corresponding fixes for CDI or Interceptors.  So, I have updated this PR to exclude EJB and EL from the CDI dependency.